pkg/server/sandbox_run.go

```diff
@@ -607,8 +604,13 @@ func hostPrivilegedSandbox(config *runtime.PodSandboxConfig) bool {
 func (c *criService) getSandboxRuntime(config *runtime.PodSandboxConfig) (criconfig.Runtime, error) {
 	untrusted := false
 	if untrustedWorkload(config) {
-		// TODO(random-liu): Figure out we should return error or not.
-		if hostPrivilegedSandbox(config) {
+		//  If the workload is requesting access to the host/node, this request will fail when using
+		// an untrusted runtime. Return error in this case.
```

Comment:
The cri-o trusted/untrusted runtime is confusing.

`If the untrusted workload is requesting access`, and remove `untrusted runtime`.

Reply:
Agreed.  I better understand the containerd approach now, and prefer this explicit behavior.